src/errors.rs

```diff
@@ -124,3 +127,31 @@ impl From<std::str::Utf8Error> for PaperoniError {
         PaperoniError::with_kind(ErrorKind::UTF8Error(err.to_string()))
     }
 }
+
+#[derive(Debug, Error)]
+pub enum LogError {
+    #[error(transparent)]
+    FlexiError(#[from] FlexiLoggerError),
+    #[error("Wrong log directory: {0}")]
```

Comment:
Change this to just:
```rust
#[error("{0}")]
```

Reply:
there is no contract that obliges the code that uses this type of error to be about the directory. are you sure you need to reduce the context? maybe I need to correct the error texts in the call?

Reply:
If you can use a different text for the context then it can stay. I think "Wrong log directory" is a little misleading since one of the possible full errors would be:
`Wrong log directory: Unable to get user directories for logging purposes` which sounds incorrect

Reply:
I thought it could be done a little better, given the small number of call places. Please check

Reply:
Looks good now.